### ISSUE TITLE
Incorrect role as link is defined for the left navigation

### ISSUE THREAD

**Issue by sebastienlevert** [storybookjs/storybook#16940]
**Describe the bug**
We have a bug that is described here https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/1092. 

The left menu of all the stories do not announce that the top level items are "dropdowns" with sublinks. 

**To Reproduce**
  **Test Environment:**   

- Browser: Microsoft edge Version 89.0.63 (Official Build) (64-bit) 
- URL:  https://mgt.dev/
- OS Version: Windows 10 Version 20H2 (OS Build 19042.685) 
- Tools used and their versions: screen reader – Narrator.

**Repro Steps:**  

1. Open the URL: https://mgt.dev/ in edge browser                         
2. Use tab to navigate to the elements which is present under components on the left side pane.                                                                                           
3. verify whether the screen reader is properly announcing the role or not. 

**Actual Result:**  
In-appropriate role as link is defined for the drop down buttons which is present under components.

**Expected Result:** 
Proper role as drop down button should be defined  for elements which is present under components as it opens pop up in that page .

**User Impact:**  
PWD user having dependency on screen reader will be confused or unable to guess the functionality and behaviour of UI component if role is not announced properly.        

https://user-images.githubusercontent.com/33054846/117261279-9c302000-ae6d-11eb-8b75-81bc980e1af4.mp4

This can be tested on this environment : https://mgt.dev/next/?path=/story/components-mgt-agenda--simple

**Comment by shilman** [comment]
It looks like this has already been fixed in Storybook. Closing this for now. Please let me know if there's more to be done here!
